src/modules/Layout/Footer/Footer.js

```diff
@@ -5,14 +5,12 @@ import './Footer.css';
 
 const b = block('j-footer');
 
-const Footer = () => {
-  return (
-    <footer className={b}>
-      <p>
-        <strong>Janus</strong> by <a href="http://hellofresh.com">HelloFresh</a>. The source code is licensed <a href="http://opensource.org/licenses/mit-license.php">MIT</a>.
-      </p>
-    </footer>
-  );
-};
+const Footer = () => (
+  <footer className={b}>
```

Comment:
just took a look at files and found out that there is a **lot** of them with the same issue...
I need to fix them all